lisa/node.py

```diff
@@ -157,8 +156,13 @@ def _check_sudo_available(self) -> bool:
         process = self._execute("ls", shell=True, sudo=True, no_info_log=True)
         result = process.wait_result(10)
         if result.exit_code != 0:
-            self.log.debug("node doesn't support sudo /bin/sh.")
-            return False
+            self.log.debug(
+                f"_check_sudo_available: 'sudo /bin/sh ls' not succeeded,stderr: {result.stderr}"
+            )
+            if "not allowed" in result.stderr:
+                # e.g. OS Security: "user is not allowed to execute '/bin/sh -c"
+                self.log.debug("node doesn't support sudo /bin/sh.")
```

Comment:
Here’s a clearer version:
"The command sudo /bin/sh -c ls may fail due to SELinux policies that restrict the use of sudo in combination with /bin/sh."